# Defaults, usually overridden in launch or specific node namespaces
wheel_controller_node:
  ros__parameters:
    kp: 2.0
    ki: 0.5
    kd: 0.1
